src/Scan/RestScans.ts

```diff
@@ -117,15 +120,28 @@ export class RestScans implements Scans {
     };
   }
 
-  private exploreDiscovery(body: ScanConfig): Discovery[] {
+  private async exploreDiscovery(body: ScanConfig): Promise<Discovery[]> {
     const discoveryTypes: Discovery[] = [];
+    const { fileId } = body;
```

Comment:
Use destructuring assignment for both `fileId` and `crawlerUrls`
 https://github.com/NeuraLegion/nexploit-cli/pull/329#discussion_r891392153